src/grpc.ts

```diff
@@ -347,26 +378,38 @@ export class GrpcClient {
    * @param {number} options.port - The port of the service.
    * @param {grpcTypes.ClientCredentials=} options.sslCreds - The credentials to be used
    *   to set up gRPC connection.
+   * @param {string} defaultServicePath - The default service path.
    * @return {Promise} A promise which resolves to a gRPC stub instance.
    */
-  async createStub(CreateStub: typeof ClientStub, options: ClientStubOptions) {
+  async createStub(
+    CreateStub: typeof ClientStub,
+    options: ClientStubOptions,
+    customServicePath?: boolean
```

Comment:
Can you add the same parameter to `createStub` in `src/fallback.ts`? It will be unused there, but we try to keep interfaces similar. Otherwise, I expect [this code](https://github.com/googleapis/nodejs-speech/blob/master/src/v1/speech_client.ts#L217-L225) in the generated client will start failing since the `createStub` there comes either from here or from the fallback code.